www/running.inc

```diff
@@ -11,7 +11,9 @@
             $noanalytics = true;
             $page_keywords = array('WebPageTest','Website Speed Test','Page Speed');
             $page_description = "Running website performance test.";
+	    
             include('head.inc');
+	    require_once WWW_PATH . '/templates/layouts/google-tag-manager.inc';
```

Comment:
@mjkozicki there is some css loaded after the gtm that is conflicting with it, moving after it fix the issue